cinder/volume/drivers/vmware/vmdk.py

```diff
@@ -1641,6 +1641,7 @@ def _in_use(self, volume):
         return (volume['volume_attachment'] and
                 len(volume['volume_attachment']) > 0)
 
+    @utils.trace
```

Comment:
What does this do?

Reply:
if debugging is on, it logs the input params and output return values for calls on the function.   it helps debugging.